src/services/document-service.js

```diff
@@ -66,6 +66,28 @@ class DocumentService {
     return documentsMetadata.sort(by(doc => doc.updatedBy, 'desc'));
   }
 
+  async getTopDocumentTags({ maxCount = 0 } = { maxCount: 0 }) {
+    const conditions = [{ roomId: null }];
```

Comment:
I think we also need `archived: false` here, right?

Reply:
Check! ✅